mmv1/third_party/terraform/website/docs/r/compute_security_policy.html.markdown

```diff
@@ -280,9 +280,31 @@ The following arguments are supported:
     * HTTP_HEADER: The value of the HTTP header whose name is configured under "enforceOnKeyName". The key value is truncated to the first 128 bytes of the header value. If no such header is present in the request, the key type defaults to ALL.
     * XFF_IP: The first IP address (i.e. the originating client IP address) specified in the list of IPs under X-Forwarded-For HTTP header. If no such header is present or the value is not a valid IP, the key type defaults to ALL.
     * HTTP_COOKIE: The value of the HTTP cookie whose name is configured under "enforceOnKeyName". The key value is truncated to the first 128 bytes of the cookie value. If no such cookie is present in the request, the key type defaults to ALL.
+    * HTTP_PATH: The URL path of the HTTP request. The key value is truncated to the first 128 bytes
+    * SNI: Server name indication in the TLS session of the HTTPS request. The key value is truncated to the first 128 bytes. The key type defaults to ALL on a HTTP session.
+    * REGION_CODE: The country/region from which the request originates.
 
 * `enforce_on_key_name` - (Optional) Rate limit key name applicable only for the following key types: HTTP_HEADER -- Name of the HTTP header whose value is taken as the key value. HTTP_COOKIE -- Name of the HTTP cookie whose value is taken as the key value.
 
+* `enforce_on_key_configs` - (Optional, [Beta](https://terraform.io/docs/providers/google/guides/provider_versions.html)) If specified, any combination of values of enforce_on_key_type/enforce_on_key_name is treated as the key on which ratelimit threshold/action is enforced. You can specify up to 3 enforce_on_key_configs. If enforce_on_key_configs is specified, enforce_on_key must not be specified. Structure is [documented below](#nested_enforce_on_key_configs).
+
+  **Note:** To avoid the conflict between `enforce_on_key` and `enforce_on_key_configs`, the field [`enforce_on_key`](#enforce_on_key) needs to be an empty string.
+
```

Comment:
Could you add an example for the scenario to the examples section in the top? Since it's a strange use case I would like to have both scenarios convered with some sort of baseline sample.

Reply:
Hi there! I put the example below the `enforce_on_key_configs` notes, I think it makes sense show the example in the same context situation instead of putting on section in the top.

What do you think about it ?

Reply:
Sorry, I would prefer the examples to be on top. It aligns with the president set in our existing documentation!! Hope that is alright !

Reply:
Don't worry :D ! I already removed and put on top, I added a little description to explain about the conflict.